src/app/app.component.spec.ts

```diff
@@ -1,25 +1,25 @@
-import {async, TestBed} from "@angular/core/testing";
+import { TestBed, waitForAsync } from "@angular/core/testing";
```

Comment:
import does not exactly match format. let's overlook that..

Reply:
how adbout the tslint rule:
```
"ordered-imports": [
			true,
			{
				"import-sources-order": "lowercase-last",
				"named-imports-order": "lowercase-first"
			}
],
```

Reply:
I meant the bracket spacing.. but nvm it! 👍🏼